lib/services/docker.pm

```diff
@@ -36,14 +36,11 @@ sub full_docker_check {
 
     check_service();
 
-    # Test the connectivity of Docker containers
-    $engine->check_containers_connectivity();
+    # Test the basic functionality
+    runtime_smoke_tests(runtime => $engine);
```

Comment:
Need include  runtime_smoke_tests before run it.

Reply:
I'm sorry. It's fixed now. Can you rerun the VR?

Reply:
Ok, re-run it, http://openqa.nue.suse.com/tests/8206368#live